Configuration/Services.yaml

```diff
@@ -34,3 +47,7 @@ services:
   JWeiland\Telephonedirectory\Repository\Handler\AddSubjectFieldToEmployee:
     tags:
       - name: 'telephonedirectory.add_record_to_employee'
+
+parameters:
```

Comment:
I really appreciate to move parameter declarations to top, behind _defaults or into a separate file.